test/passing/tests/js_source.ml.ocp

```diff
@@ -2603,11 +2599,9 @@ let f (Aux x) =
   | Succ (Succ Zero) -> "2"
   | Succ (Succ (Succ Zero)) -> "3"
   | Succ (Succ (Succ (Succ Zero))) -> "4"
-  | _ -> .
+  | _ -> . (* error *)
 ;;
 
-(* error *)
```

Comment:
hmm this seems like a regression.
A comment alone on its line marks a section and should not be aggregated to the previous match no?

Reply:
Not in this case because the original code to be formatted was:
```ocaml
let f (Aux x) =
  match x with
  | Succ Zero -> "1"
  | Succ (Succ Zero) -> "2"
  | Succ (Succ (Succ Zero)) -> "3"
  | Succ (Succ (Succ (Succ Zero))) -> "4"
  | _ -> .  (* error *)
;;
```